### PR TITLE
Features/enhanced readme

### PR DESCRIPTION
**Pull request by mayaalsamarrai**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,84 +2,88 @@
...
 
-* Name: display task name
-* Path to images: assets path to the folder with images. Clicking the folder button will open the asset search blade where you can select a folder you want to run the task in. Each task can reference only one folder.
-* Settings: you can select a number of options for this task
+* **Name**: displays the task name
+* **Path to images**: assets path to the folder with images. Clicking the folder button will result in opening the asset search blade where you can select a folder you want to run the task . Each task can refer only to one folder.
+* **Settings**: you can select a number of options for this task
 
-To create an option click on edit button next to Thumbnail settings:
+In order to create an option you should click on the 'Edit' button located next to the Thumbnail settings:
```
Comment (eMazeika) [inline README.md]: 'Edit' button not visible in the picture. Change

File: README.md [hunk 2]
```diff
@@ -2,84 +2,88 @@
...
 
 ![image](https://user-images.githubusercontent.com/20122385/36429547-a431bbfe-165b-11e8-9607-291cc511878f.png)
 
-* Name: display setting name
-* Thumbnail file name suffix: The suffix that will be added to the back of the original file name, separated by '_', for example for 'large' and the original file 
+* **Name**: display setting name
+* **Thumbnail file name suffix**: The suffix that will be added to the back of the original file name, separated by '_', for example for 'large' and the original file
```
Comment (eMazeika) [inline README.md]: change "back" to "end"

File: README.md [hunk 3]
```diff
@@ -2,84 +2,88 @@
...
+
 ```
-  https://virtocommercedemo1.blob.core.windows.net/catalog/1428965138000_1133723.jpg will be 
+  https://virtocommercedemo1.blob.core.windows.net/catalog/1428965138000_1133723.jpg will be
```
Comment (eMazeika) [inline README.md]: move "will be" out of the code section

File: README.md [hunk 4]
```diff
@@ -2,84 +2,88 @@
...
 ```
 
-* Width: Thumbnail width in pixels
-* Height: Thumbnail height in pixels
-* Resize method.
+* **Width**: Thumbnail width in pixels
+* **Height**: Thumbnail height in pixels
+* **Resize method**.
 
 # Resize method options
```
Comment (eMazeika) [inline README.md]: '###

File: README.md [hunk 5]
```diff
@@ -2,84 +2,88 @@
...
-Resize the image until one of the sides will match to the given dimensions while maintaining the aspect ratio. Part of the image will be cut off. The thumbnail will be resized and cropped the anchor position. You can specify an anchor position in the o'sption Anchor Position: TopLeft, TopCenter, TopRight, CenterLeft, Center, CenterRight, BottomLeft, BottomCenter, BottomRight.
+This method resizes the image until one of the sides matches the given dimensions while maintaining the aspect ratio. A part of the image will be cut off. The thumbnail will be resized and cropped according to the anchor position. You can specify an anchor position in the o'sption Anchor Position: TopLeft, TopCenter, TopRight, CenterLeft, Center, CenterRight, BottomLeft, BottomCenter, BottomRight.
 
 Now you can use the two thumbnails wherever you want to display a smaller version of the original image.
 You can do it, just add a needed suffix (grande or medium) to product image url.
 
-# Running the task
+## Running the task
 
 You can select multiple task and run them in the task list or run just one task from the task blade.
```
Comment (eMazeika) [inline README.md]: multiple task**s**

File: README.md [hunk 6]
```diff
@@ -2,84 +2,88 @@
...
 ![image](https://user-images.githubusercontent.com/20122385/36430796-b68279c6-165e-11e8-8e02-ae75e7a11f66.png)
 
 Available modes:
 
-* Regenerate - thumbnails will be generated for all images. 
-* Process changes - tries to find changed or added images and generates thumbnails only for them. Also will detect added options. Already generated thumbnails for removed options will not be deleted. 
+* **Regenerate** - thumbnails will be generated for all images.
+* **Process changes** - tries to find changed or added images and generates thumbnails only for them. Also will detect added options. Thumbnails generated for removed options will not be deleted.
 
 # Installation
```
Comment (eMazeika) [inline README.md]: correct link on line 138 is: https://admin-demo.virtocommerce.com/docs/ui/index#/Image%2520tools%2520module

File: README.md [hunk 7]
```diff
@@ -1,90 +1,96 @@
...
 
-# VirtoCommerce.ImageTools
-VirtoCommerce.ImageTools module represents functionality for working with images.
-Key features:
-* Make image thumbnails in different ways and use they, instead original images. For example, in listing or for previews.
-* Group image resize options into tasks and run them against an asset catalog in the background 
-* All image formats supported
+## Key features
+
+1. Possibility to make different image thumbnails and use them, instead of using the  original images. For example,this functionality can be used in listing or for previews;
```
Comment (mvktsk) [inline README.md]: 'For example, this functionality ...'
missing space after comma

File: README.md (hunk 5 above)
Comment (eMazeika) [reply README.md]: `## Running the task
is ok. What I meant was "s" missing in "You can select multiple task and"

File: README.md [hunk 8]
```diff
@@ -1,90 +1,96 @@
...
-* Process changes - tries to find changed or added images and generates thumbnails only for them. Also will detect added options. Already generated thumbnails for removed options will not be deleted. 
+* **Regenerate** - thumbnails will be generated for all images.
+* **Process changes** - tries to find changed or added images and generates thumbnails only for them. Also will detect added options. Thumbnails generated for removed options will not be deleted.
+
+## Installation
 
-# Installation
 Installing the module:
+
 * Automatically: in VC Manager go to Configuration -> Modules -> Image tools module -> Install
```
Comment (eMazeika) [inline README.md]: Automatically: in VC Manager go to More -> Modules -> Available (n) -> Image tools module -> Install

File: README.md [hunk 9]
```diff
@@ -1,90 +1,96 @@
...
 * Manually: download module zip package from https://github.com/VirtoCommerce/vc-module-image-tools. In VC Manager go to Configuration -> Modules -> Advanced -> upload module package -> Install.
```
Comment (eMazeika) [inline README.md]: Change Configuration to More

**Pull request merged**